packages/plugin/webview/src/lib/connector.ts

```diff
@@ -167,10 +168,33 @@ async function listenOnThemeChanged(client: PluginClient) {
     return cssLink;
   }
 
+  const setAttribute = (url, backupUrl = null) => {
+    // there is no way to know if it will load unless it's loaded first
+    axios.get(url).then(() => {
+        getLink().setAttribute('href', url);
+    }).catch(() => {
+        if(backupUrl) getLink().setAttribute('href', backupUrl);
+    });
+  }
+
   // If there is a url in the theme, use it
   const setLink = (theme: Theme) => {
     if (theme.url) {
-      getLink().setAttribute('href', theme.url.replace(/^http:/,"").replace(/^https:/,""))
+      const url = theme.url.replace(/^http:/, "protocol:").replace(/^https:/, "protocol:");
```

Comment:
maybe that doesn't really apply in your case, but why not always load using https?

Reply:
1. because of localhost development. the https css is not there. It used to be that the theme url was always loaded, so it was not a problem but then running plugins on the web with a http host always failed. 2. and if the host is on http and plugin too we won't be sure the https css is there too.

Reply:
I m not sure whether this can help the PR but we could also use the public http link `https://remix.ethereum.org/assets/css/...`
your solution is ok though too

Reply:
yes that would help but from the perspective of an agnostic web view, the engine can be anything, for example vscode. the code from francois was really neutral, give me url, I will add it. you could argue that is the best solution and the host should take care of the urls being all https. ... I don't think my solution is very elegant.